### ISSUE TITLE
Wifi Module

### ISSUE THREAD

**Issue by Tom2Drake** [Jutta-Proto/protocol-bt-cpp#1]
Hey,

first of all you did a stunning Job and a very good documentation. I just wonder if the Wi-Fi dongle from Jura works the same beside from the connection type? Did you ever had a look into this?

**Comment by COM8** [comment]
Yes, it works more or less the same but over wifi. It has it's own protocol from what I've found while reverse engineering the android app.
Once I have a completely working Bluetooth lib, I will most likely have a look at the wifi dongle and try to provide a lib for him as well.

Besides that, I'm currently integrating this lib into the [gtk-ui](https://github.com/Jutta-Proto/gtk-ui/tree/bt).

**Comment by Tom2Drake** [comment]
Thanks for the quick Feedback, im looking forward to see how the wifi connection is handled. My goal is to integrate it to my smart home, therefore I prefer wifi over bluetooth.

**Comment by COM8** [comment]
![signal-2022-02-15-114819_001](https://user-images.githubusercontent.com/11741404/154047640-1d82cb51-92f8-4eac-bd23-dcc27e653544.jpeg)

Got access to a WIFI-Module, but since it only supports:
* E8 (EB)
* GIGA 6 (EA)
* ENA 4 (EA)
* Z10 (EA)
* E4 (EA)

I'm not able to test it on my E6. So I just dumped the ESP32 on it and since it's not encrypted I'm now working on a bit of documentation how to dump and analyse it.

**Comment by u20p17** [comment]
Hello @COM8!
i just bought a z10 (with the wifi module) and now found this repo. Can you explain the status of it? Could I somehow help you with anything? (i am not a real programmer but can understand code) 
Thanks!

**Comment by COM8** [comment]
@u20p17 sadly I moved on from it since I do not have a coffee maker any more right now.
